### ISSUE TITLE
WCAG 2.2 3.2.6 Consistent Help understanding - remove explanation of help mechanisms

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#1881]
The whole understanding document https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html includes a whole lengthy section explaining what the various help mechanisms should provide (and because those aren't the focus of the SC, they're couched in weak "should" statements - see https://github.com/w3c/wcag/issues/1874)

That's completely out of scope for the SC itself, so the understanding document - by focusing on those help systems, rather than the actual "meat" of the SC which is the consistent location of the link/access TO help - becomes very confusing and ambiguous for auditors who're just after information relating to "what do i need to do to pass/fail this SC?".

While interesting, these various explanations/descriptions of what different forms help can take etc are orthogonal to the SC - most of this should be removed, in my view.

> Self help methods beyond the site, such as using internet search to find the contact information for an organization, can be too difficult. Further, the user’s disability may make it more difficult to find the help available (such as a “contact us” link, phone number, or support page) if the information is not present within a few interactions (e.g., displayed in the header, or via a menu). In addition, for some users with disabilities, struggling to complete a task on a site may cause additional cognitive challenges when searching for help within the site.
>
>When a user is quickly able to find help, they are able to complete the task even if they encounter challenges.
>
>At least one of the following mechanisms to get help should be included on all websites:
>
>* Human contact details such as a phone number, email address, hours of operation.
>* Human contact mechanism such as a messaging system, chat client, contact form, social media channel.
>* Self-help option such as an up-to-date Frequently Asked Questions, How Do I page, Support page.
>* A fully automated contact mechanism such as a chatbot.
>
>While it would be best for all sites to offer at least one of these types of help, this SC does not require help be provided. The SC requires that when help is available that it be in a consistent location.
>
>The human contact details should enable the user to connect with the organization or the part of the organization that can assist with the content. For example, an online jobs / recruitment portal should provide a contact method for the team that supports the recruitment portal and not a catch-all for the entire company. Each layer of contact added prolongs the time before the user will receive help.
>
>The human contact mechanism enables a person to express what they are looking for using their own words. For some with cognitive disabilities, this may be the best way for them to find an answer to their problem.
>
>For pages for which no human support is available, a self-help option should say that no human support is available. Self-help options should go beyond allowing the user to search within the site. Contextual help is still recommended (See SC 3.3.5 for more information) but a self-help option should provide a single location that makes it easier for people with cognitive disabilities to understand what help is available without having to hunt for it. While some people may easily be able to identify that no support would be available for a particular type of Web site, this may not be apparent to some users with disabilities.
>
>If a chatbot is provided, it should meet other WCAG Success Criteria for your conformance level. Chatbots which work more effectively for everyone, and particularly for people with cognitive disabilities should:
>
>* recognize misspelled words,
>* provide human contact details if the chatbot is unable to provide a satisfactory response after 3 attempts, and
be dismissed with a single interaction, and recalled using a link or button.
```

**Comment by DavidMacDonald** [comment]
We could provide the information in a separate section of best practices in the understanding to distinguish it from the strict explanation of the SC. It looks like useful information to help forward the COGA cause. Historically the Understanding docs have included supplementary information around the requirement to help authors determine the spirit of the sc.

**Comment by patrickhlauke** [comment]
Keep them in the understanding, sure, but then move them out of the "Intent" section (as they're really asides / extra information that has nothing to do directly with the SC itself). And avoid the use of "should" (even if lowercased) so as not to give the impression that these are some sort of requirements to meet the SC.

**Comment by DavidMacDonald** [comment]
agreed
